*** ./smali/android/app/DialogFragment.smali	2015-07-14 16:18:25.557731377 +0800
--- /home/mtjo/MIUI/omni-i605/temp/new_smali/framework.jar.out/./smali/android/app/DialogFragment.smali	2015-07-14 16:19:12.245729871 +0800
***************
*** 421,426 ****
--- 421,430 ----
      return-void
  
      :cond_1
+     iget-object v2, p0, Landroid/app/DialogFragment;->mDialog:Landroid/app/Dialog;
+ 
+     if-eqz v2, :cond_0
+ 
      invoke-virtual {p0}, Landroid/app/DialogFragment;->getView()Landroid/view/View;
  
      move-result-object v1
