*** ./smali/android/app/FragmentManagerImpl$6.smali	2015-11-07 12:38:56.332038920 +0800
--- /home/mtjo/patchrom/omni_i605_v7/temp/new_smali/framework.jar.out/./smali/android/app/FragmentManagerImpl$6.smali	2015-11-07 12:41:36.576041518 +0800
***************
*** 49,54 ****
--- 49,60 ----
  
      iget-object v0, p0, Landroid/app/FragmentManagerImpl$6;->val$finalFragment:Landroid/app/Fragment;
  
+     iget-boolean v0, v0, Landroid/app/Fragment;->mHidden:Z
+ 
+     if-eqz v0, :cond_0
+ 
+     iget-object v0, p0, Landroid/app/FragmentManagerImpl$6;->val$finalFragment:Landroid/app/Fragment;
+ 
      iget-object v0, v0, Landroid/app/Fragment;->mView:Landroid/view/View;
  
      const/16 v1, 0x8
