*** ./smali/android/net/IConnectivityManager.smali	2015-11-07 12:39:30.408039473 +0800
--- /home/mtjo/patchrom/omni_i605_v7/temp/new_smali/framework.jar.out/./smali/android/net/IConnectivityManager.smali	2015-11-07 12:42:14.012042125 +0800
***************
*** 447,456 ****
--- 447,472 ----
      .end annotation
  .end method
  
+ .method public abstract startUsingNetworkFeatureMSim(ILjava/lang/String;Landroid/os/IBinder;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract stopUsingNetworkFeature(ILjava/lang/String;)I
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract stopUsingNetworkFeatureMSim(ILjava/lang/String;I)I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
