*** ./smali/com/android/internal/statusbar/IStatusBarService.smali	2016-01-04 14:06:01.893995655 +0800
--- /home/mtjo/patchrom/omni_i605/temp/new_smali/framework2.jar.out/./smali/com/android/internal/statusbar/IStatusBarService.smali	2016-01-04 14:09:53.509999410 +0800
***************
*** 179,184 ****
--- 179,192 ----
      .end annotation
  .end method
  
+ .method public abstract setStatus(ILandroid/os/IBinder;Ljava/lang/String;Landroid/os/Bundle;)V
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract setSystemUiVisibility(II)V
      .annotation system Ldalvik/annotation/Throws;
          value = {
