*** ./smali/android/net/wifi/WifiConfiguration$1.smali	2015-11-07 12:39:29.496039458 +0800
--- /home/mtjo/patchrom/omni_i605_v7/temp/new_smali/framework.jar.out/./smali/android/net/wifi/WifiConfiguration$1.smali	2015-11-07 12:42:13.004042109 +0800
***************
*** 87,92 ****
--- 87,116 ----
  
      iput-object v2, v0, Landroid/net/wifi/WifiConfiguration;->preSharedKey:Ljava/lang/String;
  
+     invoke-virtual {p1}, Landroid/os/Parcel;->readString()Ljava/lang/String;
+ 
+     move-result-object v2
+ 
+     iput-object v2, v0, Landroid/net/wifi/WifiConfiguration;->wapiASCert:Ljava/lang/String;
+ 
+     invoke-virtual {p1}, Landroid/os/Parcel;->readString()Ljava/lang/String;
+ 
+     move-result-object v2
+ 
+     iput-object v2, v0, Landroid/net/wifi/WifiConfiguration;->wapiUserCert:Ljava/lang/String;
+ 
+     invoke-virtual {p1}, Landroid/os/Parcel;->readString()Ljava/lang/String;
+ 
+     move-result-object v2
+ 
+     iput-object v2, v0, Landroid/net/wifi/WifiConfiguration;->wapiPsk:Ljava/lang/String;
+ 
+     invoke-virtual {p1}, Landroid/os/Parcel;->readInt()I
+ 
+     move-result v2
+ 
+     iput v2, v0, Landroid/net/wifi/WifiConfiguration;->wapiPskType:I
+ 
      const/4 v1, 0x0
  
      .local v1, "i":I
